test/mocks/mocks.go

```diff
@@ -492,3 +565,32 @@ func ConvertToUnstructured(cr interface{}) (*unstructured.Unstructured, error) {
 	data, err := runtime.DefaultUnstructuredConverter.ToUnstructured(cr)
 	return &ustrv1.Unstructured{Object: data}, err
 }
+
+// DatabaseCRMockWithAnnotation based on PostgreSQL operator, returning a instantiated object.
+func DatabaseCRMockWithAnnotation(ns, name string) *pgv1alpha1.Database {
+	return &pgv1alpha1.Database{
```

Comment:
I fail to find the annotation in the defined value; shouldn't it be somewhere in here?

Reply:
Well, this CR mock is not really required. I will remove it.
The annotation should be there in CRD mock.